src/Common/src/TypeSystem/Ecma/EcmaMethod.cs

```diff
@@ -415,5 +416,33 @@ public override PInvokeMetadata GetPInvokeMethodMetadata()
 
             return new PInvokeMetadata(moduleName, name, (PInvokeAttributes)import.Attributes);
         }
+
+        public override PInvokeParameterMetadata[] GetPInvokeParameterMetadata()
+        {
+            if (!IsPInvoke)
```

Comment:
Why the check for IsPInvoke?

E.g. I would want to use this to implement [this TODO](https://github.com/dotnet/corert/blob/e87dc86d7f2ee32d8be1577a8410746264fd0d20/src/ILCompiler.Compiler/src/Compiler/MetadataGeneration.cs#L231).